### PR TITLE
Feature/format decimal

### PR DESCRIPTION
**Pull request by stevenJhonGil**
### **Format Decimal Numbers**
```
-Added decimal conversion to base 2 format
-Added decimal conversion to base 8 format
-Added decimal conversion to base 16 format
-Added documentation for each methods
-Added round method and its documentation
```
###### **_java Calculate round (value) (value)_**

### CODE REVIEW

**Comment by michaelcapuno** [comment]
has conflicts

**Comment by stevenJhonGil** [comment]
I have resolved the conflicts

**Comment by catalino-dev** [comment]
@stevenJhonGil :+1: This PR looks great

**Comment by kana-ph** [comment]
@stevenJhonGil 

Please add round function:

```bash
$ java Compute round 56.6667 2
56.67
```

Look at the `Math` class.

**Review by michaelcapuno — APPROVED** [review]

**Comment by stevenJhonGil** [comment]
@michaelcapuno @saviercoo I have added the round method with its Documentation

File: CalculateAction.java [hunk 1]
```diff
@@ -59,6 +59,13 @@ public String calculate(String[] args) {
 		public String calculate(String[] args) {
 			return Integer.toHexString(Integer.parseInt(args[0]));
 		}
+	},
+	ROUND {
+		@Override
+		public String calculate(String[] args) {
+			double factor = Math.pow(10,Double.valueOf(args[1]));
```
Comment (catalino-dev) [inline CalculateAction.java:66]: Amazing usage of Math.pow in finding it's factor. :+1:

**Review by catalino-dev — APPROVED** [review]

**Comment by michaelcapuno** [comment]
@saviercoo round method works :+1:

**Pull request merged**